mage_ai/frontend/pages/datasets/[slug].tsx

```diff
@@ -190,14 +244,119 @@ function Data() {
 
   const headEl = (
     <FlexContainer alignItems="justify-right" flexDirection="row-reverse" >
-      <Button 
-        onClick={viewColumns}
-      >
+      <Button onClick={viewColumns}>
         <Text bold> Column view </Text>
       </Button>
     </FlexContainer>
   );
 
+  type ActionProps = {
+    idx: number;
+    link?: () => void;
+    name: string;
+    numFeatures: number;
+    onClose: () => void;
+    showIdx?: boolean;
+  }
+
+  const Action = ({
+    idx,
+    link,
+    name,
+    numFeatures,
+    onClose,
+    showIdx,
+  }: ActionProps) => (
+    <RowCard
+      columnFlexNumbers={[0.5, 0.5, 12]}
+    >
+      {link &&
+        <Link
+          bold
+          noHoverUnderline
+          onClick={link}
+        >
+          Apply
+        </Link>
+      }
+      {showIdx && <Text>{idx+1}</Text>}
+      <FlexContainer>
+        <Text>{name},</Text>
+        <Spacing mr={1} />
+        <Text secondary>{pluralize("feature", numFeatures)}</Text>
+      </FlexContainer>
+      <FlexContainer>
+        {/* TODO: add View Code & Preview here */}
+        <Button
+          basic
+          iconOnly
+          onClick={onClose}
+          padding="0px"
+          transparent
+        >
+          <Close muted />
+        </Button>
+      </FlexContainer>
+    </RowCard>
+  );
+
+  const actionsEl = (
+    actions.map((action, idx) => {
+      const {
+        suggestions: {
+          title,
+          action_payload: {
+            action_arguments,
+          },
+        },
+      } = action;
+      const numFeatures = action_arguments.length;
+
+      return (
+        <Action
+          idx={idx}
+          key={`${idx}-${title}`}
+          name={title}
+          numFeatures={numFeatures}
+          onClose={() => removeAction(idx)}
+          showIdx
+        />
+      );
+    })
+  );
+
+  const suggestionsEl = (
+    <Accordion>
+      <AccordionPanel
+        noBackground
+        noPaddingContent
+        title={`${suggestions.length} suggested actions`}
+      >
+        {
+          suggestions.length > 0
+          ?
+          suggestions.map((suggestion, idx) => {
+            const { action_payload: { action_arguments }} = suggestion;
+            const numFeatures = action_arguments.length;
+
+            return (
+              <Action
+                idx={idx}
+                key={`${idx}-${suggestion.title}`}
+                link={() => addAction(idx)}
+                name={suggestion.title}
+                numFeatures={numFeatures}
+                onClose={() => removeSuggestion(idx)}
+              />
+            )
+          })
+          :
+          <>{/* TODO: what do we render when no suggestions exist? */}</>
```

Comment:
Would be similar to the empty datasets table. Would just say something like "No suggestions available."
<img width="645" alt="image" src="https://user-images.githubusercontent.com/78053898/170894838-c177ddc4-7a6d-4bdb-b70d-71206ff46ec4.png">